web/modules/custom/server_general/src/ThemeTrait/PersonCardsThemeTrait.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\server_general\ThemeTrait;
+
+use Drupal\server_general\ThemeTrait\Enum\AlignmentEnum;
+use Drupal\server_general\ThemeTrait\Enum\BadgeColorEnum;
+use Drupal\server_general\ThemeTrait\Enum\FontSizeEnum;
+use Drupal\server_general\ThemeTrait\Enum\FontWeightEnum;
+use Drupal\server_general\ThemeTrait\Enum\TextColorEnum;
+
+/**
+ * Helper methods for rendering Person card elements.
+ */
+trait PersonCardsThemeTrait {
+
+  use ElementLayoutThemeTrait;
+  use ElementWrapThemeTrait;
+  use InnerElementLayoutThemeTrait;
+  use CardFooterButtonThemeTrait;
+  use CardThemeTrait;
+
+  /**
+   * Build Person cards element.
+   *
+   * @param string $title
+   *   The title.
+   * @param array $body
+   *   The body render array.
+   * @param array $items
+   *   The render array built with
+   *   `ElementLayoutThemeTrait::buildElementLayoutTitleBodyAndItems`.
+   *
+   * @return array
+   *   The render array.
+   */
+  protected function buildElementPersonCards(string $title, array $body, array $items): array {
+    return $this->buildElementLayoutTitleBodyAndItems(
+      $title,
+      $body,
+      $this->buildCards($items),
+    );
+  }
+
+  /**
+   * Build a People card.
+   *
+   * @param array $person
+   *   An array containing person data.
+   *
+   * @return array
+   *   The render array.
+   */
+  protected function buildElementPersonCard(array $person): array {
```

Comment:
`array $person` isn't aligned with the rest of the code base - there's no type safety here